src/assets/webpub.js

```diff
@@ -173,3 +173,83 @@ const processFB2 = doc => {
         ]
     }
 }
+
+const webpubFromComicBookArchive = async (uri, inputType) => {
+    const cbArchiveName = decodeURI(uri).split('/').pop().split('.').slice(0, -1).join('')
+
+    let pages
+    switch (inputType) {
+        case 'cbz': pages = await unpackCBZ(uri); break
+        case 'cbr': pages = await unpackCB(uri, inputType); break
+        case 'cb7': pages = await unpackCB(uri, inputType); break
+        case 'cbt': pages = await unpackCB(uri, inputType); break
+    }
+
+    const stylesheet = `
+        * {
+            margin: 0 !important;
+            padding: 0 !important;
+        }
+        
+        body {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        }
+
+        .image-wrapper {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            
+            min-height: 99.5vh;
+        }
+
+        .image-wrapper img {
+            width: 100vw;
+            max-height: 99.5vh;
+            object-fit: contain;
+        }
+    `
+    const stylesheetBlob = new Blob([stylesheet], { type: 'text/css' })
+    const stylesheetURL = URL.createObjectURL(stylesheetBlob)
+
+    const sectionLinkObjects = pages.filter(page =>
+            ['jpeg', 'png', 'gif', 'bmp', 'webp'].includes(page.type))
+        .map(page => {
+            const html = `
+            <!doctype html>
+            <html>
+                <head>
+                    <title>${page.title}</title>
+                    <link rel="stylesheet" type="text/css" href="${stylesheetURL}" />
+                </head>
+
+                <body>
+                    <section class="image-wrapper">
+                        <img src="${URL.createObjectURL(page.blob)}" alt="${page.title}" />
+                    </section>
+                </body>
+            </html>
+            `
+
+            const pageHTMLBlob = new Blob([html], { type: 'text/html' })
+            const pageURL = URL.createObjectURL(pageHTMLBlob)
+
+            return {
+                href: pageURL,
+                type: 'text/html',
+                title: page.title
+            }
+        })
+
+    return {
+        metadata: {
+            title: cbArchiveName
```

Comment:
Add `layout: 'pre-paginated'` to render it as a fixed-layout EPUB.

Reply:
I think I have found a better way to get predictable results. Let me know if you have any ideas.

Reply:
Nice. But I think `layout: 'pre-paginated'` and `spread` should be turned on for the "Automatic" layout?

Reply:
Yes, you're right. The **Automatic** and **Single Column** layouts shouldn't have the same effect in the first place. So, this means,
  - If the selected layout is **Automatic**,
    ```js
    return {
        metadata: {
            title: cbArchiveName,
            layout: 'pre-paginated',
        },
        ...
    }
    ```
  - Else,
    ```js
    return {
        metadata: {
            title: cbArchiveName
        },
        ...
    }
    ```

Reply:
You can use `layout: 'reflowable'` for the other layouts.

Reply:
Ok, I think `reflowable` was the default option or something but it is better to explicit.

Reply:
> But, not disabling `spread` gives unpleasant results.

What do you mean by "unpleasant results"? The whole point of `layout: 'pre-paginated'` is so that it would show a two-page view, so `spread` should definitely be enabled. If you mean sizing or alignment issues, I think your original flexbox stylesheet works well with this mode.

Reply:
Yes, I meant sizing and alignment issues when I said "unpleasant results". But, you're right everything looks good with the previous flexbox-based stylesheet. Thanks for making things clear.